tests/conftest.py

```diff
@@ -249,3 +251,45 @@ def persisted_channel(request):
 def test_assets_directory() -> pathlib.Path:
     """Gets path to test_assets directory."""
     return pathlib.Path(__file__).parent / "test_assets"
+
+
+@pytest.fixture(scope="package")
+def grpc_session_creation_kwargs():
+    """Gets the keyword arguments required for creating the gRPC interpreter."""
+    channel = grpc.insecure_channel(f"localhost:31763")
+    grpc_options = GrpcSessionOptions(
+        grpc_channel=channel,
+        session_name="",
+    )
+    grpc_options = GrpcSessionOptions(channel, "")
+    return {"grpc_options": grpc_options}
+
+
+@pytest.fixture(scope="package")
```

Comment:
Scope: as long as the `grpc_channel` fixture has a broad scope, it's not critical for all of the kwargs fixtures to have a broad scope. However, pytest does seem to error if a fixture with a broad scope requests a fixture with a narrow scope, so I guess it's okay for the kwargs fixtures to have a broad scope.